packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -19,9 +19,10 @@ import {join} from 'path';
 import {existsSync, promises as fsPromises} from 'fs';
 import {loadOrCreateConfig} from '../config';
 import {enUS as messages} from '../strings';
+import {getJavaHome} from '../../../../core/src/lib/jdk/JdkHelper';
```

Comment:
You are importing classes from `core` by relative path. You need to import using `@bubblewrap/core`. This is causing the build to break.

Reply:
And let the androidSdk check stay in `doctor`? it seems weird a little bit that only one part of the two will be done in `doctor` don't it?